README.md

```diff
@@ -2,4 +2,159 @@
+
+8. Leave the `stayed` property initialized to `false`. It will need to be used to hold state during a round.
+
+9. Make `mayHit` a calculated property that determines whether the player may take a new card (if the hand is not busted, is not a blackjack, and if the player has not stayed).
+
+10. Write a method called `canPlaceBet()` which takes an unsigned integer argument and returns a boolean of whether or not the player can afford the submitted bet.
+
+11. Write two methods called `didWin()` and `didLose()` which both take an unsigned integer for the value of the bet. The `didWin()` method should add the value of the bet argument to the player's wallet, while the `didLose()` method should subtract it.
+
+12. Write a method called `didPush()` which does nothing. Leave a comment within the implementation that this inaction is intentional.
+
+### `House.swift`
+
+1. Re-enable the target membership for `HouseSpec.swift` for `swift-blackjackTests`.
+
+2. Create a new class file for the `House` class. It should be a subclass of the `Player` class.
+
+3. Override the superclass's initializer to set the `wallet` property to `1000`.
+
+4. Add a calculated property `mustHit` that returns a boolean of whether the house must take a new card or not. Treat the "house rules" as staying at a score of seventeen (17).
+
+### `Dealer.swift`
+
+1. Re-enable the target membership for `DealerSpec.swift` for `swift-blackjackTests`.
+
+2. Create a new class file for the `Dealer` class. It should have four properties:
+  * a `Deck` called `deck`,
+  * a `House` called `house` and named "House",
+  * a ***`House`*** called `player` and named "Player", and
+  * an unsigned integer called `bet` which starts at zero.  
+**Note:** *In this console-version of the game, since we don't have user interface for providing input, we're going to use the `House` class's `mustHit` method for the player's decision-making also.*
+
+3. Write a method called `placeBet()` which takes an unsigned integer argument and returns a boolean of whether or not the house and the player can both afford the submitted bet. If they can, this method should record the value of the bet being placed.
+
+4. Write a method called `deal()` that deals a new round, giving two new cards each to the player and to the house.
+
+5. Write a method called `turn()` which takes a `House` argument (*since in this console version both the house and the player will be used with this method*). If the `House` object is allowed to take a card, it should be asked if it wishes to hit or stay. If the `House` object wishes to hit, the dealer should then give it a card. If the `House` object wishes to stay, then the decision to stay should be recorded.
+
+6. Write a method called `winner()` which returns a string containing the result of the round. It should:
+  * return `"push"` if both the house and player are dealt blackjack hands,
```

Comment:
i can put it in the Advanced section like it was the Objc lab, and remove the tests for it.